product_docs/docs/epas/15/installing/linux_install_details/managing_an_advanced_server_installation/modifying_the_data_directory_location.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: "Modifying the data directory location"
+---
+
+## On RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x
+
+On a RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x host, the unit file is named `edb-as-15.service` and resides in `/usr/lib/systemd/system`. The unit file contains references to the location of the EDB Postgres Advanced Server (EPAS) `data` directory. Avoid making any modifications directly to the unit file because they might be overwritten during package upgrades.
```

Comment:
I think we can make this generic by writing <x> in place of the exact version number across the topic. Thoughts?
```suggestion
On a RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x host, the unit file is named `edb-as-<x>.service` and resides in `/usr/lib/systemd/system`. The unit file contains references to the location of the EDB Postgres Advanced Server (EPAS) `data` directory. Avoid making any modifications directly to the unit file because they might be overwritten during package upgrades.
```